.prettierrc.yaml

```
@@ -0,0 +1,2 @@
+{ "printWidth": 120, "tabWidth": 2, "useTabs": false, "proseWrap": "always", "singleQuote": false }
+ 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)